src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
The Main class should not be responsible for creating figures or supplying colors. Consider moving the logic for creating and drawing figures to a separate class that encapsulates this behavior, keeping the Main class focused on starting the application.